### ISSUE TITLE
Consuming mdspan as dependency

### ISSUE THREAD

**Issue by bernhardmgruber** [kokkos/mdspan#100]
I would like to use `mdspan` in some of my cmake-based projects as a dependency. I found a conan package [here](https://conan.io/center/mdspan). I did not find a package for spack or vcpkg.

Do you have any plans for creating/maintaining packages for the aforementioned package managers?

From a quick look at the `CMakeLists.txt`, it also looks like one could include `mdspan` via `add_subdirectory`. I have not tried it though. Is this a supported/recommended way consuming `mdspan` in a downstream project? What would you recommend?

Thank you!

**Comment by bernhardmgruber** [comment]
Ok, it seems there is a vcpkg port available that is about a month old: https://github.com/microsoft/vcpkg/pull/21170.

**Comment by crtrott** [comment]
Yeah we probably should add a spack package. In principle you should be able to use it via add_subdirectory, I mostly have installed it though via raw cmake.

**Comment by tpadioleau** [comment]
If it can help I started to write a minimalist spack recipe for testing. You can find it [here](https://github.com/pdidev/spack/blob/c13cc850af8886474a9b7a256419558b89fb4a41/packages/mdspan/package.py).
